deploy/install-dev.sh

```
@@ -0,0 +1,238 @@
+name="speedtest-ex"
+
+depend() {
+    need net
+}
+
+start_pre() {
+    checkpath --directory --mode 0755 /run
+}
+
+start() {
+    ebegin "Starting ${name}"
```

Shellcheck (0.10.0)

[warning] name is referenced but not assigned.

(SC2154)